modules/sdk-coin-dot/src/dot.ts

```diff
@@ -54,9 +54,12 @@ interface RecoveryOptions {
   recoveryDestination: string;
   krsProvider?: string;
   walletPassphrase?: string;
+  startIdx?: number;
+  numIteration?: number;
```

Comment:
lets rename `numIteration` to `scan` to be consistent with existing param name.

also, rename `startIdx` to `startingScanningIndex` (this was suggested by Ben in slack messages)